### PR TITLE
Feat/dbquery

### PR DESCRIPTION
**Pull request by wdku5791**
changed test controller, schema.js, and added a queries file

### CODE REVIEW

File: server/database/queries.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
+const pgp = require('pg-promise')();
+
+module.exports = {
+  getUser: function (userId) {
+    return db.query('select * from users');
```
Comment (alisonzthu) [inline server/database/queries.js]: getUser query, seems that "WHERE id=userId" is forgotten

File: server/database/queries.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+      email:
+      type:
+    }
+    insert into users (username, first_name, last_name, email, address, type, password) values ('wdku', 'winston', 'ku', 'wdku5791@gmail.com', '471 quail drive', 'user', 'hashed')
+    */
+    return db.query('insert into users \
+      (username, first_name, last_name, email, address, type, password)\
+      values \
+      (${username}, ${first_name}, ${last_name}, ${email}, ${address}, ${type}, ${password})\
+      returning id', userObj);
```
Comment (alisonzthu) [inline server/database/queries.js]: These ${} wrapped values probably need to be wrapped into single quotes to work.

File: server/database/queries.js (hunk 1 above)
Comment (wdku5791) [inline server/database/queries.js]: i did forget. thanks

File: server/database/queries.js (hunk 2 above)
Comment (wdku5791) [inline server/database/queries.js]: these don't need to be wrapped in anything extra

**Comment by wdku5791** [comment]
ill test them first then. it requires a little extra effort

**Pull request merged**